api/application.go

```diff
@@ -185,7 +190,7 @@ func (h ApplicationHandler) Update(ctx *gin.Context) {
 // Application REST resource.
 type Application struct {
 	ID              uint     `json:"id"`
-	Name            string   `json:"name"`
+	Name            string   `json:"name" binding:"required"`
```

Comment:
Interesting that the json tag without the `omitempty` is not the same as required.